packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -190,9 +191,9 @@ export class MetricExporter implements IMetricExporter {
    */
   private async _createMetricDescriptor(metricDescriptor: OTMetricDescriptor) {
     const authClient = await this._authorize();
-    const request = {
+    const request: monitoring_v3.Params$Resource$Projects$Metricdescriptors$Create = {
```

Comment:
I personally prefer a slightly different solution that avoids using this complicated import:

```ts
const requestBody = transformMetricDescriptor(...);
return new Promise<void>((resolve, reject) => {
  MetricExporter._monitoring.projects.metricDescriptors.create(
    {
      name: `projects/${this._projectId}`,
      requestBody,
      auth: authClient,
    },
    ...
})
```

I think there are benefits to either approach, so I'm just presenting this as something to think about.

Reply:
Let me check if typescript actually complains still if I do it that way, I'll get back to you

Reply:
Fixed